dataprofiler/profilers/profile_builder.py

```diff
@@ -1208,7 +1211,11 @@ def diff(  # type: ignore[override]
         report["data_stats"] = self._profile.diff(
             other_profile._profile, options=options
         )
-        return _prepare_report(report)
+        return _prepare_report(
```

Comment:
just curious, why do we return options here if they are passed in? I wouldn't have thought to do that.